src/openzaak/utils/expansion.py

```diff
@@ -382,5 +382,11 @@ def _render_inclusions(self, data, renderer_context):
 def get_expand_options_for_serializer(
     serializer_class: Type[Serializer],
 ) -> List[tuple]:
-    choices = [(opt, opt,) for opt in serializer_class.inclusion_serializers]
+    choices = [
+        (
+            opt,
+            opt,
```

Comment:
Seems a bit excessive for black to format it over this many lines, I think if you remove the trailing comma after the second `opt` it will keep the tuple on one line

```suggestion
            opt
```